### ISSUE TITLE
NoSuchFileException on server start

### ISSUE THREAD

**Issue by SlimeDog** [montlikadani/TabList#125]
Spigot 1.15.2
TabList 5.2

On server start-up.
```
[09:34:31] [Server thread/INFO]: [TabList] Enabling TabList v5.2
[09:34:31] [Server thread/WARN]: java.nio.file.NoSuchFileException: plugins/TabList.jar
[09:34:31] [Server thread/WARN]:        at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
[09:34:31] [Server thread/WARN]:        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
[09:34:31] [Server thread/WARN]:        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
[09:34:31] [Server thread/WARN]:        at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
[09:34:31] [Server thread/WARN]:        at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:149)
[09:34:31] [Server thread/WARN]:        at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
[09:34:31] [Server thread/WARN]:        at java.base/java.nio.file.Files.readAttributes(Files.java:1763)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1225)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:727)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.zip.ZipFile$CleanableResource.get(ZipFile.java:844)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:247)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:177)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.jar.JarFile.<init>(JarFile.java:346)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.jar.JarFile.<init>(JarFile.java:317)
[09:34:31] [Server thread/WARN]:        at java.base/java.util.jar.JarFile.<init>(JarFile.java:283)
[09:34:31] [Server thread/WARN]:        at hu.montlikadani.tablist.bukkit.utils.Util.getClasses(Util.java:76)
[09:34:31] [Server thread/WARN]:        at hu.montlikadani.tablist.bukkit.commands.Commands.<init>(Commands.java:37)
[09:34:31] [Server thread/WARN]:        at hu.montlikadani.tablist.bukkit.TabList.registerCommands(TabList.java:184)
[09:34:31] [Server thread/WARN]:        at hu.montlikadani.tablist.bukkit.TabList.onEnable(TabList.java:106)
[09:34:31] [Server thread/WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[09:34:31] [Server thread/WARN]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351)
[09:34:31] [Server thread/WARN]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480)
[09:34:31] [Server thread/WARN]:        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:464)
[09:34:31] [Server thread/WARN]:        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:378)
[09:34:31] [Server thread/WARN]:        at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:457)
[09:34:31] [Server thread/WARN]:        at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:274)
[09:34:31] [Server thread/WARN]:        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:784)
[09:34:31] [Server thread/WARN]:        at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by montlikadani** [comment]
Did you modified something in the tablist jar file?

**Comment by SlimeDog** [comment]
Of course not.

**Comment by montlikadani** [comment]
Did you compressed the TabList file into ZIP?

**Comment by SlimeDog** [comment]
Of course not.

**Comment by SlimeDog** [comment]
I renamed TabList.jar to TabList-5.2.jar
That has not been an issue with previous versions. I rename all JAR files to include the version (has should developers, but that's another story).

**Comment by montlikadani** [comment]
You should rename into the original name, because I reworked the commands registering for readability. So now, it will checks the file name to get classes in the file.

**Comment by SlimeDog** [comment]
Sheesh! Why? TL is now the only plugin of 152 I test, with this requirement. This is unnecessary.

**Comment by montlikadani** [comment]
Now I'm trying to split the filename to avoid this error.

**Comment by SlimeDog** [comment]
Thank you.

**Comment by montlikadani** [comment]
https://github.com/montlikadani/TabList/commit/66ad92752f74436e10fdab1e38e22b72bb4a4e3c

**Comment by SlimeDog** [comment]
Confirmed fix in TabList 5.2.1
Thanks.
